src/components/SensorsList/index.tsx

```diff
@@ -8,7 +8,7 @@ interface SensorsListType {
 
 export const SensorsList: FC<SensorsListType> = ({ sensors }) => {
   return (
-    <ul className='flex flex-col'>
+    <ul className='flex flex-col w-[calc(100%+16px)] ml-[-8px]'>
```

Comment:
Just out of curiosity: what are these new width and margin values for?

Reply:
`w-[]` is a way from tailwindcss JIT to render arbitrary values. `calc(100%+16px)` adds 16px to a percentual 100% width. This makes it unnecessary to use the style attribute and makes leveraging the media queries and other modifiers such as `hover` possible.